scripts/browserify-config.js

```diff
@@ -0,0 +1,48 @@
+/*
+ |--------------------------------------------------------------------------
+ | Browser-sync config file
+ |--------------------------------------------------------------------------
+ |
+ | For up-to-date information about the options:
+ |   http://www.browsersync.io/docs/options/
+ |
+ | There are more options than you see here, these are just the ones that are
+ | set internally. See the website for more info.
+ |
+ |
+ */
+
+var urllib = require('url');
+var urlParse = urllib.parse;
+function isEnabled (val) {
+  val = val || '';
+  return val !== '' && val !== '0' && val !== 'false' && val !== 'off';
+}
+
+function getEnvVar (name, defaultVal) {
+  return name in process.env ? isEnabled(name) : defaultVal;
+}
+
+var opts = {
+  server: {baseDir: './'},
+  middleware: [
+    function (req, res, next) {
+      // Route `dist/aframe.js` to `build/aframe.js` so we can
+      // dev against the examples :)
+      var path = urlParse(req.url).pathname;
+      if (path.indexOf('/' + 'dist/aframe.js') !== -1) {
+        req.url = req.url.replace('/dist/', '/build/');
+      }
+      next();
+    }
+  ],
+  rewriteRules: [],
+  files: ['build/aframe.js'],
```

Comment:
the comment got removed, so I didn't see this. because watchify's output is being piped to the browser-sync command, the changes aren't getting picked up. I'm working on a diff right now; will ping you when everything's working.